test/Bin/GameTest/GameTests_1500.cpp

```diff
@@ -682,3 +682,10 @@ GAME_TEST(Issues, Issue1947) {
     EXPECT_GT(pSpriteObjects[4].containing_item.numCharges, 0);
     EXPECT_GT(pSpriteObjects[4].containing_item.maxCharges, 0);
 }
+
+GAME_TEST(Issues, Issue1990) {
+    // Test opening the Tularean Forest half-hidden chest, which generates a black potion:
+    EXPECT_NO_THROW(
+        test.playTraceFromTestData("issue_1990.mm7", "issue_1990.json");
```

Comment:
Probably worth checking that the trace enters a chest and a potion is produced too